### DISCUSSION TITLE
Not receiving "Forgot Password" confirmation email for Mastodon.social

### DISCUSSION THREAD

**Discussion by OminousCry** [mastodon/mastodon#19926] [Q&A] [answered]
Hi there!

I signed up for Mastodon a few years ago but was inactive up until yesterday (assumed my account was deactivated for inactivity cause I couldn't find it anymore), tried to register on Mastodon.social thanks to my friend's invite but it said my username was taken which shouldn't be an issue. I tried resetting my password but never got the confirmation email for the address I originally signed up.

Thanks in advance for your help!

**Comment by leeky** [comment, marked as answer]
Each Mastodon server is looked after by different people, so it's probably worth contacting the mastodon.social team for help - see https://mastodon.social/about for their contact details.

With the mass exodus of users from Twitter to Mastodon at the moment, many servers are overloaded, so it could be that you password reset email is just taking a little while to reach the top of the queue (I confirmed an account on another instance earlier, and it took 30 minutes to arrive, but YMMV).

**Reply by willyjimmy** [reply]
And 20 days later it still doesnt 'work'!

**Comment by Joyce-Stack** [comment]
I have tried a reset password several times and I am not receiving an email either.  What should I try?

**Reply by MrGeneration** [reply]
Contact the instance admin that's listed in the `/about` page of the instance affected.

**Comment by sarahbreau** [comment]
I am in the same position, and contacting the server admin has not helped (no response). I think we are just stuck here? I wish I could log in online so I could migrate to another instance.

**Reply by MrGeneration** [reply]
I'm afraid there's no migration option for you if your admin either ignores or doesn't check mails.
